src/managers/GuildApplicationCommandManager.js

```diff
@@ -119,6 +120,87 @@ class GuildApplicationCommandManager extends ApplicationCommandManager {
     );
   }
 
+  /**
+   * Add permissions to a command.
+   * @param {ApplicationCommandResolvable} command The command to edit the permissions for
+   * @param {ApplicationCommandPermissionData[]} permissions The permissions to add to this command
+   * @returns {Promise<ApplicationCommandPermissions[]>}
+   * @example
+   * // Block the @everyone role from command permissions
+   * guild.commands.addPermissions('123456789012345678', [
+   *   {
+   *     id: guild.roles.everyone.id,
+   *     type: 'ROLE',
+   *     permission: false
+   *   },
+   * ])
+   *   .then(console.log)
+   *   .catch(console.error);
+   */
```

Comment:
Does this example actually work? I don't think Discord designed it to allow `@everyone` overrides, that's what `defaultPermission` is for.